docs/getting-started/install-from-npm.md

```diff
@@ -34,3 +34,13 @@ Alternatively when only map functionality is needed, and not stock.
 var Highcharts = require('highcharts/highmaps');
 ```
 
+Installing nightly builds of Highcharts
+--------------------------------------
+Nightly builds allows the access to the next Highcharts version prior to release and testing. Currently this is available via the [highcharts-dist nightly branch](https://github.com/highcharts/highcharts-dist/tree/nightly) on Github.
+Note that we do not recommend the use of the nightly build in production environments as these **may contain bugs and are not considered stable.**
```

Comment:
`Note that we do not recommend the use of the nightly build in production environments as it **may contain bugs and is not considered stable.**`